app/src/main/java/io/appium/uiautomator2/handler/GetClipboard.java

```diff
@@ -71,4 +69,30 @@ public AppiumResponse safeHandle(IHttpRequest request) {
             return new AppiumResponse(getSessionId(request), WDStatus.UNKNOWN_ERROR, e);
         }
     }
+
+    // Clip feature should run with main thread
+    private AppiumResponse getClipboardTextResponse(final IHttpRequest request) {
+        AppiumGetClipRunnerble runnable = new AppiumGetClipRunnerble(request);
```

Comment:
I know it is outside of this PR, but can we change `AppiumGetClipRunnerble` to `AppiumGetClipRunnable`?